### ISSUE TITLE
Among Us Steam Key Issue

### ISSUE THREAD

**Issue by aleynia** [itchio/itch.io#1128]
I bought Among Us from Inner Sloth less than a week ago. I decided to claim the Steam key, and it won't let me, it says, "There are no keys available at this time, try again later". If you need any further information please let me know. I've decided not to add any username or account information unless needed for my own privacy, and safety.

**Comment by bjorn** [comment]
Probably the reason is that there are no Steam keys available at this time. Innersloth will have to add more Steam keys to itch.io so they can be claimed. If trying again later doesn't work, you should contact Innersloth about it.

**Comment by leafo** [comment]
That's correct, we've notified the developer that the Steam keys they have provided have run out and hopefully they will restock them shortly. Once they are available you will be able to claim them from your purchase page.
